src/cli/create.ts

```diff
@@ -51,6 +51,12 @@ export async function create (destDir: string | undefined, opts: { name?: string
         destDir = path.resolve(destDir);
     }
 
+    // Only allow 'safe' domain names
+    if (invalidSubdomain(subdomain)) {
+        console.log(`\n${kleur.bold().red(`Derived subdomain '${subdomain}' is not allowed, please try again.`)}`);
+        return Promise.resolve();
```

Comment:
```suggestion
        throw new Error(`Derived subdomain '${subdomain}' is not allowed, please try again.`);
```

We generally handle errors in the CLI by throwing. There's an error handler in `cli/index.ts` that formats the error message (instead of printing a developer stack trace) and exits with the right error code.